main.go

```diff
@@ -45,8 +47,9 @@ import (
 	"github.com/fluxcd/pkg/runtime/pprof"
 	"github.com/fluxcd/pkg/runtime/probes"
 
-	"github.com/fluxcd/source-controller/api/v1"
+	v1 "github.com/fluxcd/source-controller/api/v1"
```

Comment:
Why does this add an alias?

Reply:
Unclear to me, VS Code keeps adding it. I'll remove it for now as it's unrelated to the change.